utils/scale-test

```diff
@@ -359,7 +360,7 @@ def Nelder_Mead_simplex(objective, params, bounds, epsilon=1.0e-6):
 def fit_function_to_data_by_least_squares(objective, params, bounds, xs, ys):
 
     assert(len(ys) > 0)
-    mean_y = sum(ys) / len(ys)
+    mean_y = sum(ys) // len(ys)
```

Comment:
Im having some second thoughts about this part of the change.  I think that this might actually not be correct :/